setup.py

```diff
@@ -1,26 +1,35 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+import logging
 import os
 import sys
 from typing import TYPE_CHECKING
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
 if TYPE_CHECKING:
     from typing import Iterable, Set
 
 MAGPIE_ROOT = os.path.abspath(os.path.dirname(__file__))
-MAGPIE_MODULE_DIR = os.path.join(MAGPIE_ROOT, 'magpie')
+MAGPIE_MODULE_DIR = os.path.join(MAGPIE_ROOT, "magpie")
 sys.path.insert(0, MAGPIE_MODULE_DIR)
 
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
-from magpie import __meta__             # noqa: F401
+from magpie import __meta__  # isort:skip
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Import "from magpie import __meta__" should be placed at the top of the module](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)